### PR TITLE
Improvments

### PR DESCRIPTION
**Pull request by seehase**
Added UV/Solar/ET

### CODE REVIEW

**Comment by seehase** [comment]
There is still a bug if you use other languages than german
because 'extra1_temp_humidity' cannot be translated with a default value 
My fix didn't solve that problem, but if you don't have these extra data, than it works also in other languages ... If I find some more time I'll fix it
You can see the new version on my site under 
https://solar.seehausen.org/weewx/
I integrated weewx into my solar logger project
Kind regards 
Holger

**Comment by ngulden** [comment]
Hi Holger,

thank you very much for the pull request. I try to give it review the next couple days.

Best regards,
Nico

File: skins/niculskin/index.html.tmpl [hunk 1]
```diff
@@ -78,8 +78,22 @@
                         <tr><td>$niculskinLabels.conditions.wind</td><td>$current.windSpeed $current.windDir.ordinal_compass ($current.windDir)</td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_rate</td><td>$current.rainRate</td></tr>
                         <tr><td>$niculskinLabels.conditions.in_temp_humidity</td><td>$current.inTemp | $current.inHumidity</td></tr>
+                        #if $day.extraTemp1.has_data
                         <tr><td>$niculskinLabels.conditions.extra1_temp_humidity</td><td>$current.extraTemp1 | $current.extraHumid1</td></tr>
+                        #end if
+                        #if $day.extraTemp1.has_data
```
Comment (ngulden) [inline skins/niculskin/index.html.tmpl:84]: Should this say $day.extra.Temp2.has_data?

File: skins/niculskin/index.html.tmpl [hunk 2]
```diff
@@ -78,8 +78,22 @@
...
                         <tr><td>$niculskinLabels.conditions.extra2_temp_humidity</td><td>$current.extraTemp2 | $current.extraHumid2</td></tr>
+                        #end if
+                        #if $day.UV.has_data
+                        <tr><td>UV</td><td>$current.UV</td></tr>
```
Comment (ngulden) [inline skins/niculskin/index.html.tmpl:88]: Please use a reference to the label. The english value is defined in the skin.conf starting at [line 266](https://github.com/ngulden/niculskin/blob/d534d27e9616359fa150a6fc20e1f39ca07fe9b2/skins/niculskin/skin.conf#L266). For example add a new line after line 286 with uv, et and radiation.

This way the template remains translatable and other languages can easily be added to the appropriate language skin subdirectory.

File: skins/niculskin/index.html.tmpl [hunk 3]
```diff
@@ -99,6 +113,18 @@
                         <tr><td>$niculskinLabels.conditions.barometer_min</td><td>$day.barometer.min</td><td>$day.barometer.mintime</td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_sum</td><td>$day.rain.sum</td><td></td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_rate_max</td><td>$day.rainRate.max</td><td>$day.rainRate.maxtime</td></tr>
+                        #if $day.UV.has_data
+                        <tr><td>High UV</td><td>$day.UV.max</td><td>$day.UV.maxtime</td></tr>
```
Comment (ngulden) [inline skins/niculskin/index.html.tmpl:117]: The same with the labels here. Add them to the skin.conf file and just refer to them from the template, e.g. `$niculskinLabels.conditions.uv_max`.

**Review by ngulden — CHANGES_REQUESTED** [review]
The patch looks good so far. Please see the comments and use references for the labels. This keeps the template translatable.

**Pull request closed without merging**